### ISSUE TITLE
Youtube video not downloading but is able to be watched.

### ISSUE THREAD

**Issue by RCcola1987** [yt-dlp/yt-dlp#9353]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

USA

### Provide a description that is worded well enough to be understood

When downloading the complete channel or some single videos from a creator yt-dlp errors out with "WARNING: [youtube] Skipping player response from ios client (got player response for video "M5t4UHllkUM" instead of "8swaQwN712g")" and downloads "[download] Destination: Video Not Available [8swaQwN712g].mhtml". If you go to the video in a web browser the video is available and able to be played. Also when --list-formats is used the following is outputted "
ID  EXT   RESOLUTION FPS │ PROTO │ VCODEC MORE INFO
────────────────────────────────────────────────────
sb2 mhtml 48x27        1 │ mhtml │ images storyboard
sb1 mhtml 72x45        1 │ mhtml │ images storyboard
sb0 mhtml 144x90       1 │ mhtml │ images storyboard"

the channel in question is https://www.youtube.com/@Daggerwin/videos
over 2k of his videos are having this issue. only about 1100 can be downloaded.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU "https://www.youtube.com/watch?v=8swaQwN712g"
[debug] Command-line config: ['-vU', 'https://www.youtube.com/watch?v=8swaQwN712g']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.12.30 from yt-dlp/yt-dlp [f10589e34] (pip)
[debug] Python 3.10.12 (CPython x86_64 64bit) - Linux-5.15.0-88-generic-x86_64-with-glibc2.35 (OpenSSL 3.0.2 15 Mar 2022, glibc 2.35)
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2, phantomjs 2.1.1
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2020.06.20, mutagen-1.46.0, requests-2.31.0, secretstorage-3.3.1, sqlite3-3.37.2, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1798 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2023.12.30 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2023.12.30 from yt-dlp/yt-dlp)
[youtube] Extracting URL: https://www.youtube.com/watch?v=8swaQwN712g
[youtube] 8swaQwN712g: Downloading webpage
[youtube] 8swaQwN712g: Downloading ios player API JSON
WARNING: [youtube] Skipping player response from ios client (got player response for video "M5t4UHllkUM" instead of "8swaQwN712g"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[youtube] 8swaQwN712g: Downloading android player API JSON
WARNING: [youtube] YouTube said: ERROR - Precondition check failed.
WARNING: [youtube] HTTP Error 400: Bad Request. Retrying (1/3)...
[youtube] 8swaQwN712g: Downloading android player API JSON
WARNING: [youtube] Skipping player response from android client (got player response for video "M5t4UHllkUM" instead of "8swaQwN712g"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube] Skipping player response from web client (got player response for video "M5t4UHllkUM" instead of "8swaQwN712g"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: Only images are available for download. use --list-formats to see them
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 8swaQwN712g: Downloading 1 format(s): sb0
[debug] Invoking mhtml downloader on "https://i.ytimg.com/sb/M5t4UHllkUM/storyboard3_L2/M$M.jpg?sqp=-oaymwENSDfyq4qpAwVwAcABBqLzl_8DBgi867OjBg==&sigh=rs$AOn4CLD6zz6wUMPtwBuO9zjkyt-TR6iF_g"
[download] Video Not Available [8swaQwN712g].mhtml has already been downloaded
[download] 100% of   20.07KiB
```

**Comment by bashonly** [comment]
Are you running this on a server/VPS/cloud hosting by any chance?

**Comment by dirkf** [comment]
Can't repro with yt-dlp 2023.06.22 or yt-dl master (domestic ISP).

**Comment by bashonly** [comment]
This looks like #7143, which typically only affects data center IPs. It usually means Youtube has blocked your IP. OP, are you using a VPN?

> over 2k of his videos are having this issue. only about 1100 can be downloaded.

When you say this, do you mean you already downloaded 1100 videos and then the issue manifested itself? That could mean Youtube blocked you after too many requests

**Comment by RCcola1987** [comment]
None of these. This is on my home internet connection. only certain videos are having this issue. I have tried with and without cookies. If it is a block wouldn't that block for ALL videos and not just some?

**Comment by bashonly** [comment]
This could be instead related to #9316 which was fixed by 413d3675804599bc8fe419c19e36490fd8f0b30f

@RCcola1987 Run `python3 -m pip install -U --pre yt-dlp` to update to nightly, and try again. 

> Also no issue watching on same system in browser.

In private browsing tab too? (not logged in)

**Comment by RCcola1987** [comment]
Ok so in a private window it says the video is not available. Is their a way to be logged in with ytdlp?

**Comment by RCcola1987** [comment]
yt-dlp -vU "https://www.youtube.com/watch?v=8swaQwN712g"
[debug] Command-line config: ['-vU', 'https://www.youtube.com/watch?v=8swaQwN712g']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2024.03.02.232720 from yt-dlp/yt-dlp-nightly-builds [6ad11fef6] (pip)
[debug] Python 3.10.12 (CPython x86_64 64bit) - Linux-5.15.0-88-generic-x86_64-with-glibc2.35 (OpenSSL 3.0.2 15 Mar 2022, glibc 2.35)
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2, phantomjs 2.1.1
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2020.06.20, mutagen-1.46.0, requests-2.31.0, secretstorage-3.3.1, sqlite3-3.37.2, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1834 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-nightly-builds/releases/latest
Latest version: nightly@2024.03.02.232720 from yt-dlp/yt-dlp-nightly-builds
yt-dlp is up to date (nightly@2024.03.02.232720 from yt-dlp/yt-dlp-nightly-builds)
[youtube] Extracting URL: https://www.youtube.com/watch?v=8swaQwN712g
[youtube] 8swaQwN712g: Downloading webpage
[youtube] 8swaQwN712g: Downloading ios player API JSON
[youtube] 8swaQwN712g: Downloading android player API JSON
WARNING: [youtube] Skipping player responses from ios/android/web clients (got player responses for video "M5t4UHllkUM" instead of "8swaQwN712g")
ERROR: [youtube] 8swaQwN712g: All player responses are invalid. Your IP is likely being blocked by Youtube
  File "/home/ryan/.local/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 732, in extract
    ie_result = self._real_extract(url)
  File "/home/ryan/.local/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 4074, in _real_extract
    webpage, master_ytcfg, player_responses, player_url = self._download_player_responses(url, smuggled_data, video_id, webpage_url)
  File "/home/ryan/.local/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 4038, in _download_player_responses
    player_responses, player_url = self._extract_player_responses(
  File "/home/ryan/.local/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 3727, in _extract_player_responses
    raise ExtractorError(

**Comment by bashonly** [comment]
`--cookies-from-browser` or `--cookies` ([more info](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#how-do-i-pass-cookies-to-yt-dlp))

Also, see important notes/workarounds for youtube cookies:
- https://github.com/yt-dlp/yt-dlp/issues/8227#issuecomment-1867184804

**Comment by RCcola1987** [comment]
But this still fails even with cookies

**Comment by bashonly** [comment]
See the linked issue and make sure that the cookies are actually working. Share log please

**Comment by RCcola1987** [comment]
Is their an ETA for the oauth2 plugin being added to yt-dlp? and yes it works ty.

**Comment by bashonly** [comment]
> Is their an ETA for the oauth2 plugin being added to yt-dlp

afaik no

> and how do i switch back to thew normal branch of yt-dlp with pip?

I wouldn't tbh. Stable release is 2+ months old at this point, and making all those requests with the no-longer-supported YT client versions may have contributed to your IP being semi-blocked in the first place

But if you must: `python3 -m pip install yt-dlp==2023.12.30`

Closing as duplicate of #7143

**Comment by spongebobandlolalo** [comment]
https://i.ytimg.com/sb/iVfcBbvMNeY/storyboard3_L2/M2.jpg
